app/_landing_pages/deck_landing.yaml

```diff
@@ -0,0 +1,225 @@
+content:
+  - row:
+      - column:
+          - type: header
+            config:
+              type: h1
+              text: "decK"
+
+  - row: 
+    - column: 
+      - type: structured_text
+        config: 
+          blocks:
+            - type: text
+              text: |
+               Manage Kong Konnect, Kong Gateway, and Kong Ingress Controller configuration declaratively
+  - row: 
+    - column:
+      - type: structured_text
+        config: 
+          blocks: 
+            - type: text
+              text: | 
+                decK is a command line tool that facilitates API Lifecycle Automation (APIOps) by offering a comprehensive toolkit of commands designed to orchestrate and automate the entire process of API delivery. APIOps involves leveraging automation frameworks to streamline and enforce best practices throughout the API lifecycle. This enables developers and operations teams to manage APIs from development through deployment, ensuring consistency, reliability, and speed in API integrations. 
+                decK is one of the many supported tools used to interact with Kong, to learn about other tools review our [tools page](/tools)
```

Comment:
Should we talk about what decK does more tangibly, in a following structured text section? That is, "decK manages config via state files etc etc.":

```
## How does decK work?

decK operates on state files. decK state files describe the configuration of Kong Gateway. State files store Kong Gateway configuration in a declarative format, letting you manage services, routes, plugins, consumers, and other entities that define how requests are processed and routed through Kong Gateway.

In the background, decK communicates to your Kong Gateways via the Admin API.
```

Or does this info belong elsewhere?

Reply:
I thin that belongs here/